std_msgs/String errType
float64 data